client/components/data/query-rewind-backup-status/index.js

```diff
@@ -11,19 +11,27 @@ import { connect } from 'react-redux';
  * Internal dependencies
  */
 import Interval, { EVERY_SECOND } from 'lib/interval';
-import { getRewindBackupProgress as getBackupProgress } from 'state/activity-log/actions';
+import { getRewindBackupProgress } from 'state/activity-log/actions';
 
 class QueryRewindBackupStatus extends Component {
 	static propTypes = {
 		downloadId: PropTypes.number,
 		siteId: PropTypes.number.isRequired,
 	};
 
+	componentWillMount() {
+		const { siteId } = this.props;
+
+		if ( siteId ) {
```

Comment:
I'm thinking that this could really use a good explanatory comment. it doesn't seem clear why this one has no `downloadId` but the other one is required. What is different about this call? Why is it different only on component mount?

Reply:
Added code and updated to do nothing if there's a download Id. We only want to run this on page load and at that moment there is not download Id.